### ISSUE TITLE
Support running inside asar packaged app

### ISSUE THREAD

**Issue by HaimBendanan** [giggio/node-chromedriver#251]
I am using node-chromedriver as part of an application that is distributed to customers.
I am building the app with electron-builder, but get the following error on startup:
```
...Error: spawn ENOENT
    at errnoException (child_process.js...
```

I somehow narrowed down the issue to being related to asar packaging. 
When packaging the app into an asar archive, the chromedriver will throw an error, but it will work as expected if not packaging as asar.
See the following links that discuss the issue:
https://www.electronjs.org/docs/tutorial/application-packaging
And here:
https://github.com/electron/electron/issues/9459

Do you think that it would be possible to make chromedriver work from within an asar archive?

**Comment by giggio** [comment]
I have no idea. I don't know asar.
The only place where we run a child process is this line:

https://github.com/giggio/node-chromedriver/blob/5e2c65c53a74a57e5884c4096c3a4d65367d1b01/install.js#L123

You could set the `force download` flag, and the whole method would be skipped, as per this line:

https://github.com/giggio/node-chromedriver/blob/5e2c65c53a74a57e5884c4096c3a4d65367d1b01/install.js#L116

See:

https://github.com/giggio/node-chromedriver#force-download

Are you able to do that?

**Comment by HaimBendanan** [comment]
I ended up not using asar at all.
I think that the problematic line was the one below, is that possible?
https://github.com/giggio/node-chromedriver/blob/5e2c65c53a74a57e5884c4096c3a4d65367d1b01/bin/chromedriver#L3

Btw, in my use case, i am shipping an app that uses the chrome driver, so I need to handle cases where users have different chrome versions, and where they update chrome once in a while.
I ended up calling the install script every time my app start, to install the appropriate chrome version
Would be great if it was a built in functionality (I had to do some workarounds to make it works)
